src/connection/Database.ts

```diff
@@ -48,7 +48,9 @@ export default class Database
         return new Promise((resolve, reject) => {
             this.pool.getConnection((error, connection) => {
                 if (error) {
-                    connection.destroy();
+                    console.log("ERROR", error);
```

Comment:
Could you please remove the console.log here, and add some curly brackets for the if statement below?

```ts
if (connection) {
  connection.destroy();
}
```

Reply:
Updated!